src/bin/radmin.c

```diff
@@ -452,16 +452,11 @@ static int cmd_terminate(UNUSED FILE *fp, UNUSED FILE *fp_err, UNUSED void *ctx,
 
 static int cmd_uptime(FILE *fp, UNUSED FILE *fp_err, UNUSED void *ctx, UNUSED fr_cmd_info_t const *info)
 {
-	struct timeval now;
+	fr_time_t now;
```

Comment:
ah and that's an fr_time_delta_t not an fr_time_t, and maybe rename it to uptime.